plugins/obs-ffmpeg/obs-nvenc.c

```diff
@@ -678,6 +683,25 @@ static bool init_encoder_base(struct nvenc_data *enc, obs_data_t *settings,
 	config->rcParams.vbvBufferSize = bitrate * 1000;
 	config->rcParams.multiPass = nv_multipass;
 
+	/* -------------------------- */
+	/* Region of interest         */
+
+	// ToDo make sure there are no side effects to setting this by default
+	config->rcParams.qpMapMode = NV_ENC_QP_MAP_DELTA;
```

Comment:
There's nothing in the documentation that suggests this won't work, but I can't test it right now. Need to make sure this doesn't have any negative impact.

(I'd like to set it by default so a ROI can be added at any point, no other encoder requires configuring this feature during init so I hope NVENC is also fine if we set this but just pass a `NULL` pointer until we actually have something)

Reply:
Confirmed on my NVIDIA machine that this does not have any adverse effect. In fact, the delta mode even works without excplicitly enabling it (if `qpDeltaMap` is non-NULL it always seems to be used, even if `qpMapMode` is left on `NV_ENC_QP_MAP_DISABLED`).